compiler/dyno/util/linters/CMakeLists.txt

```diff
@@ -17,22 +17,18 @@
 
 add_executable(fieldsUsed "fieldsUsed.cpp")
 
+# This option is set from the call to CMake from compiler/dyno/Makefile.help
```

Comment:
I think the below comment should move here and we should add a bit. I suggest:

```
# TODO: use DYNO_LLVM_COMP_ARGS DYNO_LLVM_LINK_ARGS instead since
# these should include clang arguments for CHPL_LLVM=system / CHPL_LLVM=bundled.
# TODO: don't try to `find_package` if CHPL_LLVM=none